src/measure_theory/decomposition/jordan.lean

```diff
@@ -0,0 +1,343 @@
+    have hμ₂ : (j₂.μ i).to_real = j₂.to_signed_measure (i ∩ Tᶜ),
+    { rw [to_signed_measure, sub_to_signed_measure_apply (hi.inter hT₁.compl),
+          show j₂.ν (i ∩ Tᶜ) = 0, by exact nonpos_iff_eq_zero.1
+            (hT₅ ▸ measure_mono (set.inter_subset_right _ _)),
+          ennreal.zero_to_real, sub_zero],
+      conv_lhs { rw ← set.inter_union_compl i T },
+      rw [measure_union, show j₂.μ (i ∩ T) = 0, by exact nonpos_iff_eq_zero.1
+            (hT₄ ▸ measure_mono (set.inter_subset_right _ _)), zero_add],
+      { exact set.disjoint_of_subset_left (set.inter_subset_right _ _)
+          (set.disjoint_of_subset_right (set.inter_subset_right _ _) disjoint_compl_right) },
+      { exact hi.inter hT₁ },
+      { exact hi.inter hT₁.compl } },
+    rw [← ennreal.to_real_eq_to_real (measure_lt_top _ _) (measure_lt_top _ _),
+        hμ₁, hμ₂, ← hj],
+    exact of_inter_eq_of_symm_diff_eq_zero_positive hS₁.compl hT₁.compl hi hS₃ hT₃ hST₁,
+    all_goals { apply_instance } },
+  { refine measure_theory.measure.ext (λ i hi, _),
+    have hν₁ : (j₁.ν i).to_real = - j₁.to_signed_measure (i ∩ S),
+    { rw [to_signed_measure, sub_to_signed_measure_apply (hi.inter hS₁),
+          show j₁.μ (i ∩ S) = 0, by exact nonpos_iff_eq_zero.1
+            (hS₄ ▸ measure_mono (set.inter_subset_right _ _)),
+          ennreal.zero_to_real, zero_sub],
+      conv_lhs { rw ← set.inter_union_compl i S },
+      rw [measure_union, show j₁.ν (i ∩ Sᶜ) = 0, by exact nonpos_iff_eq_zero.1
+            (hS₅ ▸ measure_mono (set.inter_subset_right _ _)), add_zero, neg_neg],
+      { exact set.disjoint_of_subset_left (set.inter_subset_right _ _)
+          (set.disjoint_of_subset_right (set.inter_subset_right _ _) disjoint_compl_right) },
+      { exact hi.inter hS₁ },
+      { exact hi.inter hS₁.compl } },
+    have hν₂ : (j₂.ν i).to_real = - j₂.to_signed_measure (i ∩ T),
+    { rw [to_signed_measure, sub_to_signed_measure_apply (hi.inter hT₁),
+          show j₂.μ (i ∩ T) = 0, by exact nonpos_iff_eq_zero.1
+            (hT₄ ▸ measure_mono (set.inter_subset_right _ _)),
+          ennreal.zero_to_real, zero_sub],
+      conv_lhs { rw ← set.inter_union_compl i T },
+      rw [measure_union, show j₂.ν (i ∩ Tᶜ) = 0, by exact nonpos_iff_eq_zero.1
+            (hT₅ ▸ measure_mono (set.inter_subset_right _ _)), add_zero, neg_neg],
+      { exact set.disjoint_of_subset_left (set.inter_subset_right _ _)
+          (set.disjoint_of_subset_right (set.inter_subset_right _ _) disjoint_compl_right) },
+      { exact hi.inter hT₁ },
+      { exact hi.inter hT₁.compl } },
+    rw [← ennreal.to_real_eq_to_real (measure_lt_top _ _) (measure_lt_top _ _),
+        hν₁, hν₂, neg_eq_iff_neg_eq, neg_neg, ← hj],
+    exact eq.symm (of_inter_eq_of_symm_diff_eq_zero_negative hS₁ hT₁ hi hS₂ hT₂ hST₂),
+    all_goals { apply_instance } }
+end
+
+@[simp]
+lemma to_signed_measure_to_jordan_decomposition_eq (j : jordan_decomposition α) :
+  (j.to_signed_measure).to_jordan_decomposition = j :=
```

Comment:
This should be in the `signed_measure` namespace I think, and called
```suggestion
lemma to_jordan_decomposition_to_signed_measure (j : jordan_decomposition α) :
  (j.to_signed_measure).to_jordan_decomposition = j :=
```
since this is `signed_measure.to_jordan_decomposition (to_signed_measure j) = j`

Reply:
Actually I guess that breaks dot notation, so nevermind

Reply:
But the trailing `_eq` is probably unecessary on both lemmas.